src/main/java/uk/gov/pay/api/model/PaymentError.java

```diff
@@ -15,7 +15,9 @@
         P0100,
         P0101,
         P0102,
-        P0103
+        P0103,
+        P0401,
+        P0498
```

Comment:
When the last pull request is done will be another one to try to do what do you say (completely agree)